src/main/java/org/fundacionjala/at15/pokemon/pokemon/Pokemon.java

```diff
@@ -0,0 +1,63 @@
+package org.fundacionjala.at15.pokemon.pokemon;
+
+import org.fundacionjala.at15.pokemon.pokemon.moves.Move;
+
+public class Pokemon {
+    private static final int MAX_QUANTITY_MOVES = 4;
+    private static final double TWENTY_PERCENT = 0.20;
+    private int maxHealthPoints;
```

Comment:
This name should be maxHitPoints according to Story1

Reply:
Done